src/pages/individual/EditIndividualMetadata.jsx

```diff
@@ -123,12 +146,78 @@ export default function EditIndividualMetadata({
           loading={loading}
           display="primary"
           onClick={async () => {
-            const currentPatchableValues = pick(defaultFieldValues, [
-              'names',
-            ]);
+            // validation
+            const requiredFieldErrors = metadata.reduce(
+              (memo, schema) => {
+                if (!schema?.required) return memo;
+
+                const formObject = schema.customField
+                  ? customFieldValues
+                  : defaultFieldValues;
+
+                if (!formObject[schema.name]) {
+                  const intlFieldName = intl.formatMessage({
+                    id: schema.labelId,
+                  });
```

Comment:
custom fields will not have a `labelId`, so we will need to fall back to the `label` property for those

Reply:
Addressed in https://github.com/WildMeOrg/codex-frontend/pull/287/commits/5ffb629e36b90928abb8b618ba8b77e53bb17914

I added `schema.label` as the fallback if `schema.labelId` is missing, but I also removed validation for custom field values because editing custom field values will be handled in a separate PR.